scripts/assemble_figure.py

```
@@ -563,6 +563,230 @@ def export_pptx(svg_path: Path, pptx_path: Path) -> None:
+    cfg_dir = _journal_config_dir(search_root or Path(__file__).parent)
+    slug = re.sub(r"[^a-z0-9]+", "-", name.strip().lower()).strip("-")
+    cfg_path = cfg_dir / f"{slug}.yaml"
+    if not cfg_path.exists():
+        available = sorted(p.stem for p in cfg_dir.glob("*.yaml") if p.stem != "README")
+        raise FileNotFoundError(
+            f"Journal config not found: {cfg_path}\n"
+            f"Available: {', '.join(available) if available else '(none)'}"
+        )
+    cfg = yaml.safe_load(cfg_path.read_text()) or {}
+    if not isinstance(cfg, dict) or "name" not in cfg:
+        raise ValueError(f"{cfg_path} is not a valid journal config")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)